### PR TITLE
byobu: add new version 5.127

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: var/spack/repos/builtin/packages/byobu/package.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ class Byobu(AutotoolsPackage):
     homepage = "http://www.byobu.co"
     url      = "https://launchpad.net/byobu/trunk/5.123/+download/byobu_5.123.orig.tar.gz"
 
+    version('5.127', '18ddaa94dd2eccd7ff771c9050001056')
```
Comment (healther) [inline var/spack/repos/builtin/packages/byobu/package.py]: can you provide the sha256 hash? You can just use the output of `spack checksum byobu@5.127`

Comment (matthiasdiener) [reply var/spack/repos/builtin/packages/byobu/package.py]: Changed, thanks. Had to checksum it manually though: 
```
$ spack checksum byobu@5.127
==> Error: `spack checksum` accepts package names, not URLs.
[same for byobu@5.123]
$ spack checksum byobu
==> Error: Could not find any versions for byobu
```

Comment (michaelkuhn) [reply var/spack/repos/builtin/packages/byobu/package.py]: FYI: It is `spack checksum package version`, not `spack checksum package@version`.

Comment (matthiasdiener) [reply var/spack/repos/builtin/packages/byobu/package.py]: that works, thanks.

**Pull request merged**